src/routes/v2/authenticated/sites.ts

```diff
@@ -184,7 +185,8 @@ export class SitesRouter {
     if (result.isOk()) {
       return res.status(200).json(result.value)
     }
-    return res.status(404).json({ message: "Unable to get staging status" })
+    logger.error({ message: result.error.message })
+    return res.status(200).json(undefined)
```

Comment:
does our frontend actually make use of this? (we went from returning `{message: string}` to `undefined`, which might have unintended consequences if FE always expects `message` to be present